Packages/com.unity.inputsystem/InputSystem/Utilities/PrimitiveValue.cs

```diff
@@ -450,7 +450,7 @@ public bool ToBoolean(IFormatProvider provider = null)
                 case TypeCode.Single:
                     return !Mathf.Approximately(m_FloatValue, default);
                 case TypeCode.Double:
-                    return NumberHelpers.Approximately(m_DoubleValue, default);
+                    return !NumberHelpers.Approximately(m_DoubleValue, default);
```

Comment:
Bonus bug fix?

Reply:
Yes, I spotted a bug we didn't even know we had!